metadata.yaml

```
@@ -4,12 +4,27 @@ summary: Agent for Jenkins, the OSS automation server.
 docs: https://discourse.charmhub.io/t/jenkins-agent-documentation-overview/3982
 maintainers:
     - launchpad.net/~canonical-is-devops
-    - Alexandre Gomes <alexandre.gomes@canonical.com>
-    - Benjamin Allot <benjamin.allot@canonical.com>
-    - David Andersson <david.andersson@canonical.com>
 issues: https://github.com/canonical/jenkins-agent-k8s-operator/issues
+source: https://github.com/canonical/jenkins-agent-k8s-operator
 description: |
-  Agent for Jenkins, the leading open source automation server. Jenkins provides hundreds of plugins to support building, deploying and automating any project.
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+  deploying and managing [Jenkins](https://www.jenkins.io/) Agent on Kubernetes, 
+  configurable to use a Jenkins charm deployed in another Juju model, or to connect 
+  to a standalone Jenkins instance.
+
+  This charm simplifies initial deployment and "day N" operations of Jenkins Agent
+  on Kubernetes. It allows for deployment on many different Kubernetes platforms, 
+  from [MicroK8s](https://microk8s.io) to [Charmed Kubernetes](https://ubuntu.com/kubernetes) 
+  and public cloud Kubernetes offerings.
+
+  As such, the charm makes it easy for those looking to take control of their own
+  Agents whilst keeping operations simple, and gives them the freedom to deploy on 
+  the Kubernetes platform of their choice.
+
+  For DevOps or SRE teams this charm will make operating Jenkins Agent simple and
+  straightforward through Juju's clean interface. It will allow easy deployment
+  into multiple environments for testing of changes, and supports scaling out for
```

```suggestion
  into multiple environments for testing changes, and supports scaling out for
```
A minor suggestion on wording :)